### DISCUSSION TITLE
Multiple servers?

### DISCUSSION THREAD

**Discussion by PrzemekSkw** [ajenti/ajenti#1416] [General]
Hi, can Ajenti monitor three servers on one dashboard?
Regards,

**Comment by kiarn** [comment]
Hi @PrzemekSkw 

Currently not. I've tested to monitor other servers running `docker` with a `ssh` connection ( I mean inside the `docker` plugin), and it works without problem. But it's only on my own installation, not usable at another level.

Supporting others server means another level of development : handle ssh config and ssh keys, manage node joining, adapt the ui, etc ...

I must say that I'm very interested into doing this, but I have to currently focus on the upgrade to Angular.

Regards

Arnaud
